var/spack/repos/builtin/packages/netlib-blas/package.py

```diff
@@ -0,0 +1,48 @@
+from spack import *
+import os
+
+
+class NetlibBlas(Package):
+    """Netlib reference BLAS"""
+    homepage = "http://www.netlib.org/lapack/"
+    url      = "http://www.netlib.org/lapack/lapack-3.5.0.tgz"
+
+    version('3.5.0', 'b1d3e3e425b2e44a06760ff173104bdf')
+
+    variant('fpic', default=False, description="Build with -fpic compiler option")
+    variant('fortran', default=False, description="Build with Fortran support")
+
+    # virtual dependency
+    provides('blas')
+
+    # Doesn't always build correctly in parallel
+    parallel = False
+
+    def patch(self):
+        os.symlink('make.inc.example', 'make.inc')
+
+        mf = FileFilter('make.inc')
+        if '+fortran' in spec:
+            mf.filter('^FORTRAN.*', 'FORTRAN = f90')
+            mf.filter('^LOADER.*',  'LOADER = f90')
+        mf.filter('^CC =.*',  'CC = cc')
+
+        if '+fpic' in self.spec:
```

Comment:
in description it says that `+fpic` only adds `fpic`, but you also change optimization flags, or it appears to be so. Perhaps introduce something like `opts=['-O2','-frecursive']`, then if `+fpic`, add `-fpic` (this is actually compiler-dependent!). Finally do filtering unconditionally. Same for CFLAGS.